client/state/notices/actionCreators.js

```diff
@@ -0,0 +1,27 @@
+import { createNoticeAction, removeNoticeAction } from './actions';
+
+export default function( dispatch ) {
```

Comment:
I'm not sure I like the naming of this file. First, it doesn't adhere to our naming convention of using dash delimiters, not camel-case. More importantly, the functions in `actions.js` are truly action creators themselves. The function here is something like an action creator factory (action creator creator? :smile: ). Personally, I'd prefer that we simplify to a single `createNotice` asynchronous action creator in `actions.js`, with bound convenience action creators. If you wanted, you could still choose to split off the synchronous action creator, though I don't see the benefit of doing so.

```jsx
// client/state/notices/actions.js

export function createNotice( status, text, options ) {
	return function( dispatch ) {
		dispatch( { /* ... */ } );

		setTimeout( () => {
			dispatch( { /* ... */ } );
		}, options.duration );
	}
}

export const createSuccessNotice = createNotice.bind( null, 'is-success' );
```